plugin.yaml

```diff
@@ -147,6 +146,21 @@ node_types:
             cloudstack_config:
                 default: {}
 
+    cloudify.cloudstack.nodes.Volume:
+            derived_from: cloudify.nodes.Volume
+            properties:
+                volume: {}
+                use_external_resource:
```

Comment:
where are  you using this property? 
'create' should read this and not do anything if its set to true.